WireMock/wwwroot/scripts/MatcherUI/MatcherUI.ts

```
@@ -141,9 +148,61 @@ export default class MatcherUI {
+
+            // Body-Scrolling deaktivieren
+            document.body.classList.add('modal-open');
+            document.body.style.overflow = 'hidden';
+            document.body.style.paddingRight = '17px'; // Kompensiert die Scrollbar-Breite
+        }
+
+        // Optional: Hinzufügen von Event-Listenern für das Schließen
+        const closeButtons = errorModal?.querySelectorAll('[data-bs-dismiss="modal"]');
+        closeButtons?.forEach(button => {
+            button.addEventListener('click', () => {
+                this.closeErrorModal();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)